glossary.html

```diff
@@ -31,12 +31,10 @@ <h1>Acronym Glossary</h1>
     </main>
     <footer>
         <hr>
-         <span class="byline">
-            <blockquote cite="https://examples.yourdictionary.com/examples-of-haiku-poems.html"><br /> Hello!
-                <br>
-                <em><b>&mdash; Matsuo Basho</b></em>
-            </blockquote>
-        </span>
+        <blockquote cite="https://examples.yourdictionary.com/examples-of-haiku-poems.html"><br /> Hello!
+            <br>
+            <span class="byline"><em><b>&mdash; Matsuo Basho</b></em></span>
+        </blockquote>
```

Comment:
the `<span>` needs to go inside the `<blockquote>` because `<span>` is an inline element and `<blockquote>` is a block element